src/resources/de/strings.json

```diff
@@ -0,0 +1,21 @@
+{
+  "languages":{
+    "en-US":{
+      "damage_reply":"Operator $author, the damage flowchart, at your request.",
```

Comment:
"damage_reply":"Operator $author, der Schadensgraph, wie angefragt."

Reply:
if you want to contribute changes, submit them on crowdin, which is where these are automatically generated from